### DISCUSSION TITLE
Fixed two bugs in drawLine that causes it to never use WebGl

### DISCUSSION THREAD

**Discussion by Jesper-FRVR** [KilledByAPixel/LittleJS#39] [General]
function drawLine(posA, posB, thickness=.1, color, useWebGL=glEnable)
{
    const halfDelta = vec2((posB.x - posA.x)/2, (posB.y - posA.y)/2);
    const size = vec2(thickness, halfDelta.length()*2);
    drawRect(posA.add(halfDelta), size, color, halfDelta.angle(), useWebGL);
}

**Comment by Jesper-FRVR** [comment]
sry for not making a pull request ... but I've never figured it out :(

and thanks for a great library btw <3

**Comment by KilledByAPixel** [comment]
thanks, i got it fixed! the =glEnable isn't necessary because it will eventually default to that in drawTile, but the extra 0,s was definitely a bug.
